pkgs/data/documentation/cheat/default.nix

```diff
@@ -0,0 +1,19 @@
+{ python35Packages, fetchurl, lib }:
```

Comment:
It's better to not fix to python-3.5. `python3Packages` would be a better choice.